test/testdata/multiple-storage-classes/local-path-config.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: local-path-config
+  namespace: local-path-storage
+data:
+  config.json: |-
+    {
+            "storageClassConfigs":{
+                   "local-path": {
+                          "nodePathMap": [
+                          {
+                                  "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
+                                  "paths":["/opt/local-path-provisioner"]
+                          }
+                          ]
+                   },
+                   "local-path-shared": {
+                          "sharedFilesystemPath": "/opt/local-path-provisioner-shared"
+                   }
+            }
+    }
```

Comment:
https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.26/deploy/local-path-storage.yaml using 
```
  config.json: |-
    {
            "nodePathMap":[
            {
                    "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
                    "paths":["/opt/local-path-provisioner"]
            }
            ]
    }
```

Do you suggest using
```
  config.json: |-
    {
            "storageClassConfigs":{
                   "local-path": {
                          "nodePathMap": [
                          {
                                  "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
                                  "paths":["/opt/local-path-provisioner"]
                          }
                          ]
                   }
            }
    }
```
is better?

The we can deprecate the old confg.

Reply:
Not sure. It might make sense to leave it as-is so that existing installations don't change unexpectedly (even if the behavior is identical), and using the new config would be left as opt-in.